builtin_metadata.json

```diff
@@ -15898,6 +15899,31 @@
     },
     "wasm": false
   },
+  "strings.render_template": {
+    "args": [
+      {
+        "description": "a templated string",
+        "name": "value",
+        "type": "string"
+      },
+      {
+        "description": "a mapping of template variable keys to values",
+        "name": "vars",
+        "type": "object[string: any]"
+      }
+    ],
+    "available": [
+      "edge"
+    ],
+    "description": "Renders a templated string with given template variables injected. For a given template string and key/value mapping, values will be injected into the template where they are referenced by key.",
```

Comment:
Maybe we should link to the go text/template [docs](https://pkg.go.dev/text/template) here, as the word `template` doesn't give many clues to what that actually entails.

Reply:
The change would need to go into `ast/builtins.go`, of course 😄 .